addons/im_livechat/static/tests/new/livechat_service_tests.js

```diff
@@ -22,3 +29,21 @@ QUnit.test("persisted session history", async (assert) => {
     const { root } = await start();
     assert.containsOnce(root, ".o-mail-Message:contains(Old message in history)");
 });
+
+QUnit.test("previous operator used when available", async (assert) => {
+    const pyEnv = await startServer();
+    const channelId = await loadDefaultConfig();
+    const [channel] = pyEnv["discuss.channel"].searchRead([["id", "=", channelId]]);
+    setCookie("im_livechat_previous_operator_pid", JSON.stringify(channel.livechat_operator_id[0]));
```

Comment:
Note that `OPERATOR_COOKIE` could be exported and imported there, so you don't have to hardcode this technical string all the time (and as long as it's exists and referred to correct expected cookie entry, that's what matters)

Reply:
Keeping this in mind in case we need it elsewhere someday